be/src/vec/aggregate_functions/aggregate_function_linear_histogram.cpp

```diff
@@ -0,0 +1,60 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
```

Comment:
please add some regress tests

Reply:
![6ad57bf705237c195f6150709752c37](https://github.com/user-attachments/assets/1d4c5241-bdaa-4979-b3b3-64d8779e3727)
Hello, do I need to manually add sql in each of these files?

Reply:
No need to add every file.
You can refer another agg function.